api.md

```diff
@@ -1832,31 +1832,68 @@ DROP MATERIALIZED VIEW contagg_view;
 
 ## refresh_continuous_aggregate() :community_function: [](refresh_continuous_aggregate)
 
-Refresh all buckets of a continuous aggregate between two points of
+Refresh all buckets of a continuous aggregate between two points in
 time. 
 
-The function expects the parameter values to have the same time type
-as used in the continuous aggregate's time bucket expression (e.g., if
-the time bucket specifies in `timestamptz`, then the start and end time
-supplied should also be `timestamptz`).
+The refresh will only materialize at the granularity of a bucket, as
+determined by the continuous aggregate view query's `time_bucket`
+expression. Therefore, only buckets that are completely covered by the
+_refresh window_ will be updated, and the window should account for
+any time zone shifts when applicable (note that a continuous aggregate
+using the `DATE` or `TIMESTAMP WITH TIME ZONE` type aligns with the
+UTC time zone).
```

Comment:
"Granularity" is a little to esoteric and suggest to use plural form for "continuous aggregates" (since we use that elsewhere). Also, parentheses have a tendency to clutter the text and are usually avoided or at least limited to very small phrases. An attempt at a rewrite below, but not entirely happy about it so feel free to tweak it.

```suggestion
Refresh process entire buckets and not partial buckets. This means that only buckets that are completely covered by the refresh window will be updated, and the window should account for
time zone shifts (when applicable). Note that continuous aggregates
using the `DATE` or `TIMESTAMP WITH TIME ZONE` type align with the
UTC time zone.
```
It is also not very clear what "aligns with the [...] time zone" means, so maybe just mention that or say that `DATE` and `TIMESTAMP WITH TIME ZONE` works/do not works/something else.

Reply:
@mkindahl  I don't understand what you are suggesting in your last paragraph.

Reply:
I am not sure "granularity" is a particularly difficult word to understand, especially for a developer audience. Still, I will try to avoid this term.